net/bwping/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=bwping
-PKG_VERSION:=2.5
+PKG_VERSION:=2.6
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=@SF/bwping
```

Comment:
Do you still use sourceforge? Because I checked URL below, which redirected me to github.io  and yours webpage's download leads to GitHub.

Reply:
Hi @BKPepe yes, both SouceForge and GitHub can still be used to download packages, app website has been moved to GitHub Pages, but there is a proper redirect from the SourceForge web hosting to GitHub pages, so all SourceForge-related URLs should still work just fine.